internal/livelint/check_can_access_app.go

```diff
@@ -81,7 +81,8 @@ func (n *Livelint) canPortForward(pod apiv1.Pod, port int32, checkFunc func(uint
 	for range readyChan {
 	}
 
-	n.ui.Send(showSpinnerMsg{showing: true})
+	oldUI, _ := n.ui.(*BubbleteaUI)
```

Comment:
I feel the spinner would also fit in the UI interface, as "active waiting" is definitely a state you can also show in a pure CLI implementation (i.e. adding dots periodically or updating the output with a / - \ | sequence).

Maybe: `StartSpinner` and `StopSpinner` would be possible additions to the UI. A very basic implementation can then just ignore this request, but it would be nice if all our UI implementations would let the user know if something is happening. 

https://odino.org/command-line-spinners-the-amazing-tale-of-modern-typewriters-and-digital-movies/

Reply:
Agreed. Good point! Added `StartSpinner` and `StopSpinner`.